From f23b4008f46685f3e318fd285665b76df4664b07 Mon Sep 17 00:00:00 2001
From: Michael Catanzaro <mcatanzaro@redhat.com>
Date: Fri, 5 Sep 2025 14:13:54 -0500
Subject: [PATCH] Don't install extra library files

We're installing a few things too many:

/usr/include/gnome-mines_gnome-games-support/libgnome-games-support.h
/usr/lib64/libgnome-games-support-3.a
/usr/share/vala/vapi/gnome-games-support-3.vapi

This made sense when libgnome-games-support was a system library
(although of course it used to be a .so rather than a .a library).
Doesn't make sense anymore.

Also, fix whitespace of variables defined right below.
---
 games/meson.build | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/games/meson.build b/games/meson.build
index 76ef49d..f750fb4 100644
--- a/games/meson.build
+++ b/games/meson.build
@@ -18,16 +18,10 @@ lggs = static_library('gnome-games-support-3',
         'scores/dialog.vala',
         'scores/score.vala',
     ],
-    install: true,
-    install_dir: [
-        true, # default libdir
-        join_paths(includedir, includesubdir), # headers
-        true, # default vapidir
-    ],
     c_args: [
         '-w',
-	'-DGETTEXT_PACKAGE="@0@"'.format (gettext_package),
-	'-DLOCALEDIR="@0@"'.format (localedir),
+        '-DGETTEXT_PACKAGE="@0@"'.format (gettext_package),
+        '-DLOCALEDIR="@0@"'.format (localedir),
     ],
     vala_header: 'libgnome-games-support.h',
     dependencies: lggs_interface_deps,
-- 
GitLab

